setup.cfg

```
@@ -58,6 +58,8 @@ cncf.kubernetes =
 databricks =
     apache-airflow-providers-databricks>=2.2.0
     databricks-sql-connector>=2.0.4;python_version>='3.10'
+dbt =
```

```suggestion
dbt.cloud =
```
Similar to the comment above. It's a little misleading to say it's "dbt" when that is typically reserved for dbt Core (local use) vs. dbt Cloud (managed service) IMO.

---

Same her corrected it to `dbt.cloud`